server/.env.example

```diff
@@ -1,2 +1,6 @@
 DATABASE_URL="postgresql://user:password@localhost/database"
-DATABASE_SSL="false"
\ No newline at end of file
+DATABASE_SSL="false"
+AUTH0_SECRET=secret
```

Comment:
I noticed you had some missing env vars. You should try to keep the `.env.example` up to date to make starting from scratch easier.